service/terraform-plugin/src/main/java/com/solace/maas/ep/event/management/agent/plugin/terraform/manager/TerraformUtils.java

```diff
@@ -62,6 +65,45 @@ public static void setCommandError(Command command, Exception e) {
                 .build());
     }
 
+    /**
+     * Delete the Terraform state directory for the given request under the given directory.<br>
+     * The directory name is constructed as follows: {context}-{serviceId}
+     * @param request
+     * @param directory
+     */
+    public static void deleteConfigPath(CommandRequest request, String directory) {
```

Comment:
I was wondering if we should validate the input parameters as this is a public method
- directory cannot be blank 
- request cannot be null